src/chainparamsbase.cpp

```diff
@@ -73,30 +80,17 @@ const CBaseChainParams& BaseParams()
 std::unique_ptr<CBaseChainParams> CreateBaseChainParams(const std::string& chain)
 {
     if (chain == CBaseChainParams::MAIN)
-        return std::unique_ptr<CBaseChainParams>(new CBaseMainParams());
+        return MakeUnique<CBaseChainParams>("", 5888); //# was 8332);
     else if (chain == CBaseChainParams::TESTNET)
-        return std::unique_ptr<CBaseChainParams>(new CBaseTestNetParams());
+        return MakeUnique<CBaseChainParams>("testnet3", 15888); //# was 18332);
```

Comment:
i think this should be `testnet` and not `testnet3`

Reply:
James says we should leave this at testnet3 as that was used previously.